Adding namespace support documentation (#5197)
elastic/integrations: packages/cloud_security_posture/docs/README.md

The integration supports **elastic agent** version 8.5 and above.

> ## Namespace support
> The KSPM installation is currently only supported using the default namespace.
> 
> Failing to install the integration using the default namespace will result in a invalid integration state. 

Reviewer: suggestion:
```suggestion
## Namespace support
KSPM is currently only supported with the default namespace configuration.

Failing to install the integration using the default namespace will result in an invalid integration state. 
```
